crates/sqlexec/src/remote/planner.rs

```diff
@@ -145,9 +149,63 @@ impl ExtensionPlanner for DDLExtensionPlanner {
                 };
                 Ok(Some(Arc::new(exec)))
             }
-            ExtensionType::DropSchemas => todo!(),
-            ExtensionType::DropTunnel => todo!(),
-            ExtensionType::DropViews => todo!(),
+            ExtensionType::DropSchemas => {
+                let drop_schemas_lp = match node.as_any().downcast_ref::<DropSchemas>() {
+                    Some(s) => Ok(s.clone()),
+                    None => Err(internal!("DropSchemas::try_decode_extension failed",)),
+                }
+                .unwrap();
+
+                let exec: DropSchemasExec = DropSchemasExec {
+                    catalog_version: self.catalog_version,
+                    names: drop_schemas_lp
+                        .names
+                        .into_iter()
+                        .map(|n| n.schema_name().to_string())
+                        .collect(),
```

Comment:
Why do we need to iterate? Can we not do `drop_schemas_lp.name`?

Reply:
converting SchemaReference to String